Dockerfile

```diff
@@ -18,13 +21,13 @@ RUN poetry config virtualenvs.create false
 RUN poetry install --no-interaction --no-root
 
 # Copy files to image
-COPY ./src ./src
 COPY ./cli ./cli
 COPY ./tests ./tests
-RUN poetry install --no-interaction
+COPY ./.git ./.git
+COPY ./.pre-commit-config.yaml ./.flake8 ./.gitignore ./
 
 # Pre-download the model
 ENV PYTHONPATH "${PYTHONPATH}:/app"
 
 # Run the indexer on the input s3 directory
-ENTRYPOINT [ "sh", "./cli/run.sh" ]
+CMD [ "sh", "./cli/run.sh" ]
```

Comment:
**explain:** Modeled after https://github.com/climatepolicyradar/navigator-document-parser/blob/572fd0217abfdac81c83afcf61dbfdfcf5369ae1/Dockerfile#L36. It not being an entrypoint makes it easier to override the command.